content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -160,7 +160,7 @@ for more information.
 
 This section contains the necessary steps to install `CRI-O` as CRI runtime.
 
-Use the following commands to install CRI-O on your system:
+Use the following commands to install CRI-O on your system (but be sure the major/minor version of CRI-O matches the major/minor version of kubernetes (see the [cri-o compatiblity matrix](https://github.com/cri-o/cri-o)):
```

Comment:
Hello @larntz-tbc . Thanks for opening this PR.
What do you think about adding a note shortcode (with the compatibility information) after line 161, such as:

```
{{< note >}}
The CRI-O major and minor versions must match the Kubernetes major and minor versions.
For more information, see the [CRI-O compatiblity matrix](https://github.com/cri-o/cri-o).
{{< /note >}}

Reply:
I think that's a great idea.  I wasn't sure what the best way to format it was, but felt like the additional information was important. 

Thank you for the suggestion!

Reply:
@larntz-tbc, Yes, depending on how you created this pull request, you should create another commit on your branch (in your local repository) and push the changes again, `git push origin <your branch>`.
The Kubernetes `Intermediate Contributing` docs describes how to work locally,
https://kubernetes.io/docs/contribute/intermediate/#work-on-the-local-repository .
Let me know if you need help sorting out the process 😃 .